.github/workflows/npm-cd.yml

```diff
@@ -2,6 +2,14 @@
 
 name: Continuous Deployment
 
+env:
+  MATRIX: '[{"OS": "ubuntu","NAMED_OS": "linux","RUNNER": "ubuntu-latest","ARCH": "x64","TARGET": "x86_64-unknown-linux-gnu"},
```

Comment:
can it be json-formatted with newlines to make it more readable, similar to how it was in the 'build' section?

Reply:
I've trued it. Unfortunately the toJson and fromJson are not so sophisticated, and it get broken when theres new line in a middle of an object

Reply:
But open the file, here in the review its look less good than when you see the real file.